server/src/main/java/io/seata/server/storage/db/lock/DataBaseDistributedLocker.java

```diff
@@ -83,9 +82,10 @@ public DataBaseDistributedLocker() {
                 @Override
                 public void onChangeEvent(ConfigurationChangeEvent event) {
                     String newValue = event.getNewValue();
-                    if (StringUtils.isNotBlank(newValue) && newValue.equalsIgnoreCase(distributedLockTable)) {
+                    if (StringUtils.isNotBlank(newValue)) {
                         distributedLockTable = newValue;
                         init();
```

Comment:
https://github.com/seata/seata/pull/4086/files#diff-9d54969bbbe87110956c8290b50a5a1e562e370d376adb9f3ececd6b813a8a23R98

Where should init be called ？